src/Components/MPO.jl

```diff
@@ -160,3 +160,58 @@ function Base.rand(rng::Random.AbstractRNG, ::Type{MPO}; n, maxdim=nothing, elty
     # TODO order might not be the best for performance
     return MPO(arrays; order=(:l, :i, :o, :r))
 end
+
+"""
+    autompo_periodic(L, one_body, two_body; eltype=ComplexF64)
+
+Builds a translationally invariant MPO of length ``L`` for Hamiltonians of the form:
+
+H = sum_i alpha_i * O_i + sum_{i<j} beta_{ij} * O_i ⊗ O_j
```

Comment:
The double tick is required to be formatted with Latex, or are you avoiding that?

```suggestion
``H = \sum_i alpha_i * O_i + \sum_{i<j} beta_{ij} * O_i \otimes O_j``
```